public/locales/en-US/send.ftl

```diff
@@ -117,3 +117,146 @@ passwordIsSet = Password set
+#  notSupportedLink = Why is my browser not supported?
+#  notSupportedOutdatedDetail = Unfortunately this version of Firefox does not support the web technology that powers Firefox Send. You’ll need to update your browser.
+#  updateFirefox = Update Firefox
+#  deletePopupCancel = Cancel
+#  deleteButtonHover = Delete
+#  footerLinkLegal = Legal
+#  footerLinkPrivacy = Privacy
+#  footerLinkCookies = Cookies
+#  passwordTryAgain = Incorrect password. Try again.
+#  javascriptRequired = Firefox Send requires JavaScript
+#  whyJavascript = Why does Firefox Send require JavaScript?
+#  enableJavascript = Please enable JavaScript and try again.
+#  # A short representation of a countdown timer containing the number of hours and minutes remaining as digits, example "13h 47m"
+#  expiresHoursMinutes = { $hours }h { $minutes }m
+#  # A short representation of a countdown timer containing the number of minutes remaining as digits, example "56m"
+#  expiresMinutes = { $minutes }m
+#  # A short status message shown when the user enters a long password
+#  maxPasswordLength = Maximum password length: { $length }
+#  # A short status message shown when there was an error setting the password
+#  passwordSetError = This password could not be set
+#
+#
+#
+# vnext ->
+#
+#
+#
+introTitle = Simple, private file sharing
+introDescription = Firefox Send lets you share files with end-to-end encryption and a link that automatically expires. So you can keep what you share private and make sure your stuff doesn't stay online forever.
+notifyUploadEncryptDone = Your file is encrypted and ready to send
+archiveExpiryInfo = Expires after { $downloadCount } or { $timespan }
+timespanMinutes = { $num ->
+        [one] 1 minute
+       *[other] { $num } minutes
+    }
+timespanDays = { $num ->
+        [one] 1 day
+       *[other] { $num } days
+    }
+timespanWeeks = { $num ->
+        [one] 1 week
+       *[other] { $num } weeks
+    }
+fileCount = { $num ->
+    [one] 1 file
+   *[other] { $num } files
+}
+totalSize = Total size: { $size }
+copyLinkDescription = Copy the link to share your file:
+copyLinkButton = Copy link
+downloadTitle = Download files
+# Firefox Send is a brand name and should not be localized.
+downloadDescription = This file was shared via Firefox Send with end-to-end encryption and a link that automatically expires.
+trySendDescription = Try Firefox Send for simple, safe file sharing.
+# count will always be > 10
+tooManyFiles = { $count ->
+    *[other] Only { $count } files can be uploaded at a time.
+}
+# count will always be > 10
+tooManyArchives = { $count ->
+    *[other] Only { $count } archives are allowed.
+}
+expiredTitle = This link has expired.
+# Firefox Send is a brand name and should not be localized.
+notSupportedDescription = Firefox Send will not work with this browser. Send works best with the latest version of Firefox, and will work with the current version of most browsers.
```

Comment:
Since we're at this, we should create a term with this note

```
-send-brand = Firefox Send
```

And use it all the new strings instead of `Firefox Send` (do not modify old strings though)

```
notSupportedDescription = { -send-brand } will not work with this browser. Send works best with the latest version of Firefox, and will work with the current version of most browsers.
```